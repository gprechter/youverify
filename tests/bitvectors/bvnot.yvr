a, b, c: BV[32]
d, e: BV[4]
f: BV[7]

a = BV{3554760518, 32}
call b = bvnot(a)
call c = bvnot(BV{5, 32})
call d = bvnot(BV{0, 4})
call e = bvnot(BV{10, 4})
call f = bvnot(BV{127, 7})