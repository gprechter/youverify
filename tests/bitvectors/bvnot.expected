{
    "a": 3554760518,
    "b": 740206777,
    "c": 4294967290,
    "d": 15,
    "e": 5,
    "f": 0
}